src/ssl/ErrorDetailManager.cc

```diff
@@ -56,23 +57,24 @@ Ssl::ErrorDetailsList::getRecord(Security::ErrorCode value, ErrorDetailEntry &en
     return false;
 }
 
-const char *
+const SBuf
 Ssl::ErrorDetailsList::getErrorDescr(Security::ErrorCode value)
 {
     const ErrorDetails::const_iterator it = theList.find(value);
```

Comment:
@rousskov , I must be missing something really important. Such as.. where is theList initialised?